docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -601,6 +601,10 @@ Displays debug information in the upper right corner. This can be useful for deb
 - **None** displays tooltips only when a data point is clicked.
 - **Details** displays tooltips when a mouse pointer hovers over a data point.
 
+## Keyboard support
```

Comment:
I am unsure if this should be here, as it seems to talk more about configurations. Also, at the bottom might not be the best place? I would appreciate some feedback.

Reply:
I see what you mean. It ends up a bit buried given that there are so many configuration options. I would move it up to just after the youtube video link that's on line 49.